frontend/src/Frontend.hs

```diff
@@ -53,7 +53,7 @@ frontend = Frontend
   , _frontend_body = prerender_ loaderMarkup $ do
     (fileOpened, triggerOpen) <- openFileDialog
     let store = browserStorage
-        crypto = Crypto mkSignature (const genKeyPair)
+        crypto = Crypto mkSignature (const genKeyPair) (\_ _ -> pure $ Left "not supported")
```

Comment:
You could use Frontend.Crypto.Ed25519.deriveKeyPairFromPrivateKey here, but it probably doesn't matter since it wont get used anyway.